osu.Game.Rulesets.Osu/Scoring/OsuScoreProcessor.cs

```diff
@@ -41,6 +41,10 @@ protected override void SimulateAutoplay(Beatmap<OsuHitObject> beatmap)
                     // Ticks
                     foreach (var unused in slider.Ticks)
                         AddJudgement(new OsuJudgement { Result = HitResult.Great });
+
+                    //Repeats
+                    foreach (var unused in slider.RepeatPoints)

```

Comment:
not sure if this will be an issue, but these are being added out of order

Reply:
I actively play lazer with this and it seems to work fine, but do you want it above `ticks` anyway?

Reply:
above isn't even correct; they are interspersed. it's fine this way for now though.